manifests/init.pp

```diff
@@ -92,12 +92,9 @@
     fail('cfg parameter must be a hash')
   }
 
+  anchor { 'grafana::begin': } ->
```

Comment:
yeah, ok, if you're doing a reporter of a bug a favour, then I don't have any big problem with it.